### PR TITLE
Update checkout.md to include AMY link

### PR DESCRIPTION
**Pull request by hnorton613**
Added the AMY portal link to the "Get Involved" step for clarification on where to go.

_If this pull request addresses an open issue on the repository, please add 'Closes #NN' below, where NN is the issue number._

_Please briefly summarise the changes made in the pull request, and the reason(s) for making these changes._

_If any relevant discussions have taken place elsewhere, please provide links to these._

<details>

For more guidance on how to contribute changes to a Carpentries project, please review [the Contributing Guide](CONTRIBUTING.md) and [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html).

Please keep in mind that lesson Maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

</details>

### CODE REVIEW

File: learners/checkout.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Briefly, the three steps are:
 
 #### How will the Checkout steps be tracked?
  **For the “Get Involved” step, trainees must login to AMY to report which participation option they would like to have counted.**
- Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts.
+ Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts. You can track your progress at https://amy.carpentries.org/.
```
Comment (brownsarahm) [inline learners/checkout.md:20]:
```suggestion
 Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts. You can track your progress in [AMY](https://amy.carpentries.org/).
```

plain URLs are bad for screen readers and this is not in an exercise, so making it better

**Review by brownsarahm — COMMENTED** [review]
I would like to improve the link format to be more accessible

**Comment by ndporter** [comment]
Requesting confirmation from @sheraaronhurt or @ErinBecker that this accurately reflects what new instructor trainees will experience before merging. Thanks!

**Comment by ErinBecker** [comment]
Looks good to me! Thanks @hnorton613 and @brownsarahm.

**Pull request merged**